alf/algorithms/entropy_target_algorithm.py

```diff
@@ -240,9 +247,9 @@ def adjust_alpha(self, entropy):
         target_entropy = self._target_entropy()
 
         if target_entropy > 0:
-            fast_stage_thresh = 0.5 * target_entropy
+            fast_stage_thresh = (0.5 * target_entropy)
```

Comment:
Why this change?

Reply:
The brackets are unnecessary. They are not removed when I revert some unnecessary conversions.  Will remove them.

Reply:
Removed now.